scripts/superdesk-authoring/authoring.js

```diff
@@ -1502,7 +1505,7 @@
                 };
 
                 $scope.save_enabled = function() {
-                    return $scope.dirty || $scope.item._autosave;
+                    return (confirm.dirty = $scope.dirty || $scope.item._autosave);
```

Comment:
it's both, returns boolean and assigns it to confirm service. maybe it could done in 2 steps to make it more obvious